src/platform/monitoring/DowntimeNotification/config/services.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * A list of services that correspond to those which we have downtime information as provided by the API.
+ */
+export default {
+  appeals: 'appeals',
+  arcgis: 'arcgis',
+  dslogon: 'dslogon',
+  emis: 'emis',
+  es: 'es',
+  evss: 'evss',
+  idme: 'idme',
+  mvi: 'mvi',
+  mhv: 'mhv',
+  tims: 'tims',
+  vic: 'vic'
+};
```

Comment:
These are all of the external services with maintenance windows as defined by the API.